test/clj/game_test/cards/resources.clj

```diff
@@ -803,6 +803,22 @@
       (prompt-choice :runner "Card from hand")
       (prompt-choice :runner "Steal"))))
 
+(deftest gang-sign-correct-prompts
+         ;; Gang Sign - accessing from HQ, not including root. Issue #2113.
+   (do-game
+     (new-game (default-corp [(qty "Hostile Takeover" 1) (qty "Snare!" 1)])
+               (default-runner [(qty "Gang Sign" 1)]))
+     (take-credits state :corp)
+     (play-from-hand state :runner "Gang Sign")
+     (take-credits state :runner)
+     (play-from-hand state :corp "Hostile Takeover" "New remote")
+     (score-agenda state :corp (get-content state :remote1 0))
+     (prompt-choice :runner "Card from hand")
+     ;; Runner has "wait for Snare, wait for on-access" prompts.
+     (is (= 2 (count (:prompt (get-runner)))) "Runner only has the Waiting prompt, not Snare!'s pay-prompt")
```

Comment:
Would this not be true if Runner had pay prompt and corp had waiting prompt?

Reply:
No, the runner would have 3 prompts at that point. (Pay for Snare. Waiting for Corp to use Snare [this prompt is always shown to Runner regardless of who sees the Pay prompt]. Waiting for on-access effects to resolve.)

Reply:
Ah, I see